### PR TITLE
Disable verbose errors and add --verbose CLI flag 

### PR DESCRIPTION
**Pull request by adelhult**
Resolves GH-246. I added a `--verbose` flag as @CMDJojo suggested! : )

### CODE REVIEW

**Comment by CMDJojo** [comment]
Hmm, it would be much better to have it configurable from outside. Maybe it would be possible without too much work to take it as an argument to the `eval` function and have a CLI flag (and hard-code it in the playground until we implement a switch there)?

File: core/src/context.rs [hunk 1]
```diff
@@ -58,18 +59,17 @@ pub struct Issue {
...
-        if let Some(input) = &self.input {
-            write!(
-                f,
-                "{} -> {}: {}, input: {}",
-                self.source, self.target, self.description, input
-            )
-        } else {
-            write!(
+        match self.input.as_ref().map(String::as_str) {
+            Some("input") | None => write!(
```
Comment (CMDJojo) [inline core/src/context.rs:63]: What does this matching against `Some("input")` do? When is the `input` argument equals to `input`? I don't think you did something wrong, I'm just curious

**Review by CMDJojo — APPROVED** [review]
Good job! Will be nice to be able to skip verbose errors, they are not always very useful

File: core/src/context.rs (hunk 1 above)
Comment (adelhult) [reply core/src/context.rs:63]: Oops! I think it actually is a type and should say something along the lines of "<unknown>" and not "input". Good catch! I'll fix that before I rebase

**Pull request merged**